modules/persistent-storage-sharing-configmaps-across-namespaces.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: PROCEDURE
+
+[id="persistent-storage-sharing-configmaps-across-namespaces_{context}"]
+= Sharing a config map across namespaces
+
+[role="_abstract"]
+To share a config map across namespaces in a cluster, you create a `SharedConfigMap` object for that config map.
+
+.Prerequisites
+
+You have privileges to perform the following actions:
+
+* Create instances of the `sharedsecrets.sharedresource.openshift.io` CRD at a cluster-scoped level.
+* Manage cluster roles and role bindings to control which users can get, list, and watch those instances.
```

Comment:
since `cluster role` is a distinct thing, the way you worded it implied the use of the distince "cluster role", instead of "role" and role binding

In general, we want to promote various roles and role bindings, where are per namesapce, in the various namespaces across the cluster, vs. a single cluster role and cluster role binding

Reply:
Understood. Updated this instance here and one in modules/persistent-storage-sharing-secrets-across-namespaces.adoc, too.